python/afdko/buildcff2vf.py

```diff
@@ -484,10 +587,14 @@ def main(args=None):
         os.path.dirname(options.var_font_path), STAT_FILENAME)
     if os.path.exists(stat_file_path):
         logger.progress("Importing STAT table override...")
-        varFont.importXML(stat_file_path)
+        import_stat_override(varFont, stat_file_path)
+
+    validate_stat_axes(varFont)
+    validate_stat_values(varFont)
+    update_stat_name_ids(varFont)
 
     varFont.save(options.var_font_path)
-    logger.progress("Built variable font '%s'" % (options.var_font_path))
+    logger.progress("Built variable font '%s'" % options.var_font_path)
```

Comment:
At some point it would be good to update everything to use f-strings. This is fine for now, but the more I use f-strings the more I despise `%`-formatting and `.format()`.

Reply:
Might as well start updating now.